opensrp-chw/src/main/java/org/smartregister/chw/provider/ChwRegisterProvider.java

```diff
@@ -163,8 +163,8 @@ private UpdateAsyncTask(Context context, RegisterViewHolder viewHolder, String f
         @Override
         protected Void doInBackground(Void... params) {
             list = getChildren(familyBaseEntityId);
-            ancWomanCount = ChwApplication.ancRegisterRepository().getAncPncWomenCount(familyBaseEntityId, CoreConstants.TABLE_NAME.ANC_MEMBER);
-            pncWomanCount = ChwApplication.ancRegisterRepository().getAncPncWomenCount(familyBaseEntityId, CoreConstants.TABLE_NAME.PNC_MEMBER);
+            ancWomanCount = getPncAncWomenCount(familyBaseEntityId, true);
+            pncWomanCount = getPncAncWomenCount(familyBaseEntityId, false);
```

Comment:
This reuses the code to get the different ANC or PNC counts of women from the respective tables. 
While the idea is for the code to be reusable, unfortunately, someone has to follow the code to determine which count is returned based on a boolean value. 
I'd suggest that maybe have a function that is reused that takes the table name (String) from which to return the count, then call the function from different methods e.g. 
`getAncWomenCount(entityId)`
`getPncWomenCount(entityId)`

One of the functions
`... getAncWomenCount(entityId){ return getWomenCount("anc_table", "entity_id"); }`
... A reusable function that takes the table name and entity id
`getWomenCount(tablename, entityId){
select count() from tablename where entityId = entityId; 
}
`